data/web/js/site/mailbox.js

```diff
@@ -601,6 +601,14 @@ jQuery(function($){
           defaultContent: '',
           className: 'none'
         },
+        {
+          title: lang.allow_aliases_wildcard,
+          data: 'allow_wildcard_aliases',
+          defaultContent: '',
+          render: function (data, type) {
+            return 1==data?'<i class="bi bi-check-lg text-warning"><span class="sorting-value">1</span></i>':(0==data?'<i class="bi bi-x-lg"><span class="sorting-value">0</span></i>':2==data&&'&#8212;');
```

Comment:
Why is this code minimized?
Since this isn't generated code from a build, I assume it can be written to be more easily readable like so:
```js
function (data, type) {
  return 1 === data
    ? '<i class="bi bi-check-lg text-warning"><span class="sorting-value">1</span></i>'
    : (0 === data
      ? '<i class="bi bi-x-lg"><span class="sorting-value">0</span></i>'
      : 2 === data && '&#8212;'
    );
```
Also, I switched from "==" to "===" it would require testing

Reply:
Upon closer inspection, I would rewrite this code like so:
```js
function (data, type) {
  if (data === 0) return '<i class="bi bi-x-lg"><span class="sorting-value">0</span></i>'
  if (data === 1) return '<i class="bi bi-check-lg text-warning"><span class="sorting-value">1</span></i>'
  if (data === 2) return '&#8212;'
}
```

Reply:
It's to keep code consistency. I don't like vanilla implementation either. But I prefer a refactoring PR, rather than changing it now. What do you think about ?

Reply:
Mmh I didn't look at the other equivalent implementations in the file.
I usually like consistency but I'd prefer something a bit more readable in that case.

Reply:
Tough call D:
I'm on the project since I started reviewing your PR so, since you have more experience than me on it, I let you choose. I'm ok with both decisions ^^

Reply:
At the moment, it's more important to keep writing style consistency (because any developer who will come in the project will have to deal with this code writing style)

Reply:
:ok_hand: @lebouquetin so I resolve this conversation, thanks @Skylsmoi for your review :heart: